common/teacmd/git.go

```diff
@@ -132,3 +157,55 @@ func replaceInFile(filePath, oldStr, newStr string) error {
 	}
 	return writer.Flush()
 }
+
+// appendToToml reads the given TOML file, appends the specified section and fields, and writes it back.
+func appendToToml(filePath, section string, fields map[string]interface{}) error {
+	// Read the existing content of the TOML file
+	fileContent, err := os.ReadFile(filePath)
+	if err != nil {
+		return eris.Wrap(err, "error reading file")
+	}
+
+	// Unmarshal the file content into a map
+	var config map[string]interface{}
```

Comment:
nit: you can actually use `any` in place of `interface{}` and it looks very pretty in my opinion 😛

Reply:
updated